packages/components/src/circular-option-picker/utils.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Computes the common props for the CircularOptionPicker.
+ */
+export function useComputeCircularOptionPickerCommonProps(
```

Comment:
```suggestion
export function getComputeCircularOptionPickerCommonProps(
```

Should we use the `get` prefix instead of `use`? The latter is a reserved prefix in React for hooks, but since this method isn't a hook, it does not need to adhere to the "Rules of Hook" enforced by ESLint/React.